cvat-core/src/api.js

```diff
@@ -697,6 +697,7 @@ function build() {
              * @property {string} proxy Axios proxy settings.
              * For more details please read <a href="https://github.com/axios/axios"> here </a>
              * @memberof module:API.cvat.config
+             * @property {string} basename
```

Comment:
Document its function?

Perhaps it should be `basePath`?

Reply:
baseUrl is not accessible as env variable from where I can see, so probably have to keep them both.
Do we want to rename it to basePath? It's a bit of work as there are basename used some other dirs.